tests/tests-cwp.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -e
+
+current_directory=$(pwd)
+test_framework_directory="$current_directory/.jenkinsfile-runner-test-framework"
+working_directory="$test_framework_directory/work-cwp"
+
+version="256.0-test"
+jenkinsfile_runner_tag="jenkins-experimental/jenkinsfile-runner-test-image"
+downloaded_cwp_jar="to_update"
+
+. $test_framework_directory/init-jfr-test-framework.inc
+
+oneTimeSetUp() {
+  rm -rf "$working_directory"
+  mkdir -p "$working_directory"
+  downloaded_cwp_jar=$(download_cwp "$working_directory")
+  jfr_tag=$(execute_cwp_jar_and_generate_docker_image "$working_directory" "$downloaded_cwp_jar" "$version" "$current_directory/test_resources/cwp-produced-images/config/packager-config.yml" "$jenkinsfile_runner_tag" | grep 'Successfully tagged')
```

Comment:
I think you don't need to save the result and provide it to "execution_should_success". If you don't specify anything, it will gather the logs automatically.

Reply:
I tried but it didn't worked. I'll give it a try again

Reply:
Now I remember why I had to do it this way. The log only is created when the executed function is `run_jfr_docker_image`, that is not the case.